public/scripts/Table.js

```diff
@@ -63,27 +69,29 @@ var Rowcell = React.createClass({
 });
 
 var TableRows = React.createClass({
-    getRowkeys: function (row) {
-        for(var k in row) {
-            if (row.hasOwnProperty(k))
-                return <Rowcell cell={row[k]}/>;
 
+    getRowKeys: function (row) {
+        var cells = [];
+        for (var k in row) {
```

Comment:
this loop is run only on the `data` object..... So no `header keys` from `headers` are referred here....

this is why the result table does not have proper cells in the proper columns

```
headers.map(function(header, headerKey){
 for (var k in row) {
  if (row.hasOwnProperty(k))
   cells.push(<Rowcell cell={row[headerKey]}/>);
 }
});
```